mapbox/vector-tiles
# [docs] set siteOrigin to enable sitemap

By setting `siteOrigin` Batfish will generate a sitemap of this site. This is part of a sitemap side quest: https://github.com/mapbox/documentation/issues/128

To test this PR:

1. `npm run build`
2. `npm run serve-static`
3. Navigate to http://localhost:8080/vector-tiles/sitemap.xml  and you should see an XML document with the files in this site!
